### PR TITLE
PR: Improvements to the theme style

### LINKED ISSUE #7: Initial improvements to theme layout and styling
Hey @juanis2112 , @dalthviz and @ccordoba12 here are the improvements as discussed during the meeting:

* [x] Shift heading font sizes down one level
* [x] Make main font size 15 px
* [x] Change left sidebar heading and subhead sizes to 0.8rem
* [x] Change top-level left sidebar headings to uppercase style
* [x] 0.3em margin-bottom on all heading h2 and below
* [x] 1.5rem margin-bottom on images
* [x] Increase width of right sidebar by ~50-100px
* [x] Increase total content max width to ~1280 px

For now, it appears these are basic, general and interlinked enough that they should all be made in the theme.

Feel free to edit this OP if I've missed something. Thanks!

~~Edit:~~

~~* [ ] Add footer~~

### PR DESCRIPTION
**Pull request by dalthviz**
A preview:

image

image

Fixes #7

### CODE REVIEW

**Comment by ccordoba12** [comment]
Thanks @dalthviz for your great work! We'll review it in our Monday's meeting.

File: src/scss/theme.scss [hunk 1]
```diff
@@ -392,11 +402,18 @@ div.prev-next-bottom {
 /* Header Styling */
 
 h1 {
-  font-size: 3rem;
+  font-size: 2.5rem;
 }
 
 h2 {
-  font-size: 2.5rem;
+  font-size: 1.75rem;
```
Comment (CAM-Gerlach) [inline src/scss/theme.scss:409]: As we discussed and I included on issue #7 , I thought we agreed on sliding down all of the header sizes one level, not just h1 and h2.

File: src/scss/theme.scss [hunk 2]
```diff
@@ -392,11 +402,18 @@ div.prev-next-bottom {
...
+  margin-top: 0.3em;
```
Comment (CAM-Gerlach) [inline src/scss/theme.scss:429]: Similarly, I thought we discussed doing this for all headers below h1, not just h2.

**Review by CAM-Gerlach — COMMENTED** [review]
Just a couple comments before the meeting.

**Review by CAM-Gerlach — APPROVED** [review]
Looks good to me now, thanks @dalthviz ! I'll let @juanis2112 make the call from here.

**Review by ccordoba12 — APPROVED** [review]
I also approve these changes. Thanks a lot for the improvements @dalthviz!

**Review by juanis2112 — APPROVED** [review]
Looks good to me!

**Pull request merged**